package.json

```diff
@@ -29,7 +30,9 @@
     "prismjs": "^1.6.0",
     "prop-types": "^15.5.10",
     "react": "^15.5.4",
+    "react-autosuggest": "^9.3.1",
```

Comment:
@kentcdodds This is what makes docsearch run. Algolia Docsearch is a wrapper over react-autosuggest